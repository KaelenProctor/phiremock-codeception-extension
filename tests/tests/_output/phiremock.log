[2020-03-31 16:21:50] stdoutLogger.INFO: Starting Phiremock in debug mode... [] []
[2020-03-31 16:21:50] stdoutLogger.DEBUG: Phiremock's expectation dir: /home/mcustiel/.phiremock/expectations [] []
[2020-03-31 16:21:50] stdoutLogger.DEBUG: Registering shutdown function [] []
[2020-03-31 16:21:50] stdoutLogger.DEBUG: PCNTL present: Installing signal handlers [] []
[2020-03-31 16:21:50] stdoutLogger.INFO: Phiremock http server listening on 0.0.0.0:18080 [] []
[2020-03-31 16:21:51] stdoutLogger.INFO: Stopping Phiremock... [] []
[2020-03-31 16:21:51] stdoutLogger.INFO: Bye bye [] []
[2020-03-31 16:21:51] stdoutLogger.INFO: Stopping Phiremock... [] []
[2020-03-31 16:21:51] stdoutLogger.INFO: Bye bye [] []
