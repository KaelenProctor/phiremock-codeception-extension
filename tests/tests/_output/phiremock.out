Server running at http://0.0.0.0:18080
